api/envoy/config/trace/v2alpha/xray.proto

```diff
@@ -19,8 +19,8 @@ message XRayConfig {
   // If this value is not set, the default value of 127.0.0.1:2000 will be used.
   api.v2.core.SocketAddress daemon_endpoint = 1;
 
-  // The name of the X-Ray segment. By default this will be set to the cluster name.
-  string segment_name = 2;
+  // The name of the X-Ray segment.
+  string segment_name = 2 [(validate.rules).string = {min_len: 1}];
```

Comment:
Just wondering, would this change break anyone?

Or was it already broken for users who did not specify a segment name?